### DISCUSSION TITLE
Can I create "flat" (=non-isomeric) SMILES inside the PostgreSQL cartridge?

### DISCUSSION THREAD

**Discussion by kienerj** [rdkit/rdkit#7302] [Q&A]
I only see function `mol_to_smiles(mol) : returns the canonical SMILES for a molecule.` which does not seems to have any option to include stereochemistry or not.

So to be sure, this has to be done from application-side and can't be done in the cartridge?

**Comment by greglandrum** [comment]
@kienerj, it does seem to be impossible to generate SMILES without including stereo. It should be easy to fix that though.
Would you mind creating an issue for that?

**Reply by greglandrum** [reply]
An inefficient workaround would be to convert the molecule into SMILES, remove `@`, convert back into a molecule, and then regenerate SMILES:
```
select mol_to_smiles(mol_from_smiles(replace(mol_to_smiles(m)::text,'@',''))) from with_chirality;
```
but this is clearly not something you'd want to have in a production system without a massive `FIXIT` warning nearby.
